rhtap/summary.sh

```diff
@@ -34,6 +34,21 @@ function cosignTree() {
     cosign tree $URL
 }
 
+echo "SUMMARY_EYECATCHER_BEGIN"
+
+jq -n --arg base "$BASE_RESULTS" '
+  [ inputs ]' < <(
+    find "$BASE_RESULTS" -mindepth 2 -maxdepth 2 -name STATUS | while read -r status_file; do
+        name=$(basename "$(dirname "$status_file")")
+        result=$(tr -d '\n' < "$status_file" | xargs | tr '[:upper:]' '[:lower:]')
```

Comment:
Do we need the `tr -d '\n'` and `xargs` calls? The status files should contain just one word, right? And subshell substitution already strips trailing newlines